GitUI/UserControls/ExListView.cs

```diff
@@ -336,37 +435,10 @@ private static int GetGroupId(ListViewGroup listViewGroup)
             return -1;
         }
 
-        public void SetGroupState(ListViewGroupState state)
-        {
-            if (!EnvUtils.RunningOnWindows() || Environment.OSVersion.Version.Major < 6)
-            {
-                // Only Vista and forward
-                // allows collapse of ListViewGroups
-                return;
-            }
-
-            foreach (ListViewGroup lvg in Groups)
-            {
-                SetGrpState(lvg);
-            }
-
-            Refresh();
-            return;
+        [Category("Behavior"), DefaultValue(false)]
+        public bool AllowCollapseGroups { get; set; }
```

Comment:
please move all properties before methods, it makes it easier to maintain the code

Reply:
ok